tests/js/spec/components/organizations/getParams.spec.jsx

```diff
@@ -129,6 +129,20 @@ describe('getParams', function () {
     ).toEqual({statsPeriod: '14d'});
   });
 
+  it('should return pageStart and pageEnd even if start and end are provided', function () {
```

Comment:
@Zylphrex I think this is backwards according to what this test is:

```suggestion
  it('should return start and end even if pageStart and pageEnd are provided', function () {
```

Reply:
No, this is not backward because `pageStart` and `pageEnd` should override `start` and `end`. Though reading it now, I do agree the name of the test is ambiguous. I'll clarify.

Reply:
Ok that makes sense